### ISSUE TITLE
scope="col" is missing for th tag and scope = "row" is missing for very first td in each tr tag, in table, in Design screen.

### ISSUE THREAD

**Issue by ramdev1980** [percussion/percussioncms#882]
**Describe the bug**
scope="col" is missing for th tag and scope = "row" is missing for very first td in each tr tag, in table.

**To Reproduce**

1. Go to Design  (http://localhost:9992/cm/app/?view=design&site=test&path=%2FSites%2Ftest%2F&pathType=page)
2. Click on List View button, given in top left side of screen. It will open a table.
3. Inspect element and you will find that scope="col" is missing for th tag and scope = "row" is missing for very first td in each tr tag, in table.

**Expected behavior**
scope="col" for th tag and scope = "row" for very first td in each tr tag, in table, should be present.

**Additional context**
Navigate all the options available for similar issue and fix.
